modules/oc-mirror-v2-about.adoc

```diff
@@ -0,0 +1,65 @@
+
+
+It provides the following features:
+
+* Provides a centralized method to mirror {product-title} releases, Operators, helm charts, and other images.
+* Maintains update paths for {product-title} and Operators.
+* Uses a declarative image set configuration file to include only the {product-title} releases, Operators, and images that your cluster needs.
+* Performs incremental mirroring, which reduces the size of future image sets.
+* Prunes images from the target mirror registry that were excluded from the image set configuration since the previous execution.
+* Optionally generates supporting artifacts for OpenShift Update Service (OSUS) usage.
+* Use the existing workflow use case/s (as in v1)
+- Mirror to Disk
+- Disk to Mirror
+- Mirror to Mirror
+* Improve overall user experience
+- Use of local caching mechanism
+- Continue where you left off
+- Save time for mirroring (especially for multiple enclaves)
+- Geared for gitops (fail fast)
+* Support multi tier complex enclaves 
+- Partial or fully air-gapped
+- Hub and spoke type air-gapped environments
+- Generate archives (tar.gz diff)
+-- By time (i.e from first mirror to latest)
+-- Incremental
+- History metadata
+* OCI FBC support 
+* Filtering of operators by selecting bundles 
+* Generate all relevant metadata 
+- IDMS - ImageDigestMirrorSet
+- ITMS - ImageTagMirrorSet
+- CatalogSource
+- Use of templating
+* Optionally generates artifacts for OSUS (Openshift Update Service)
+* Implement delete functionality
+- Use of DeleteImageSetConfig (kind)
+- Delete remote registry (manifests only)
+- Delete local cache 
+
+
+When using the oc-mirror plugin, you specify which content to mirror in an image set configuration file. In this YAML file, you can fine-tune the configuration to only include the {product-title} releases and Operators that your cluster needs. This reduces the amount of data that you need to download and transfer. The oc-mirror plugin can also mirror arbitrary helm charts and additional container images to assist users in seamlessly synchronizing their workloads onto mirror registries.
+
+The first time you run the oc-mirror plugin, it populates your mirror registry with the required content to perform your disconnected cluster installation or update. In order for your disconnected cluster to continue receiving updates, you must keep your mirror registry updated. To update your mirror registry, you run the oc-mirror plugin using the same configuration as the first time you ran it. The oc-mirror plugin references the metadata from the storage backend and only downloads what has been released since the last time you ran the tool. This provides update paths for {product-title} and Operators and performs dependency resolution as required.
```

Comment:
With v2, you don't need to use the same imagesetconfig nor same metadata from storage backend ....
You can have as many imagesetconfigs as you want, targeting eventually different registries as well.
The constraint of maintaining same imagesetconfig + same storage backend + same destination registry is no longer relevant in v2.
The cache is what helps oc-mirror achieve the goal of only downloading what is missing in the destination registry, instead of what is new compared to last time you ran the tool.

Reply:
Ack